frontend/public/locales/en/public.json

```
@@ -281,6 +281,21 @@
   "ConfigMaps": "ConfigMaps",
   "ConfigMap details": "ConfigMap details",
   "Binary data": "Binary data",
+  "Required": "Required",
+  "Key must be unique": "Key must be unique",
+  "Cannot update the object when immutable field is set to true": "Cannot update the object when immutable field is set to true",
+  "Config maps hold key-value pairs that can be used in pods to read application configuration.": "Config maps hold key-value pairs that can be used in pods to read application configuration.",
+  "This object has been updated.": "This object has been updated.",
+  "Click reload to see the new version.": "Click reload to see the new version.",
+  "Save": "Save",
+  "A unique name for the configmap within the project": "A unique name for the configmap within the project",
```

we can be consistent, below I see `ConfigMap` and it make sense as we are referring to k8s resource
```suggestion
  "A unique name for the ConfigMap within the project": "A unique name for the ConfigMap within the project",
```